sky/shell/gpu/direct/rasterizer_direct.cc

```diff
@@ -66,6 +67,10 @@ void RasterizerDirect::OnAcceleratedWidgetAvailable(gfx::AcceleratedWidget widge
   glClearColor(0.0f, 0.0f, 0.0f, 1.0f);
   glClear(GL_COLOR_BUFFER_BIT);
   surface_->SwapBuffers();
+  if (did_draw) {
+    did_draw->Signal();
+    did_draw = nullptr;
```

Comment:
Yeah, it's left over from when this was a member and I was playing with different timing.  :)